timemachine/md/minimizer.py

```diff
@@ -59,15 +59,18 @@ def parameterize_system(topo, ff: Forcefield, lamb: float):
             lamb,
         ),
     ]
-    return params_potential_pairs
+    return [p for _, p in params_potential_pairs], [params for params, _ in params_potential_pairs]
```

Comment:
nit, readability: single-letter abbreviation `p` for words that both start with `p`

maybe 
```suggestion
    return [pot for (_, pot) in params_potential_pairs], [params for (params, _) in params_potential_pairs]
```
or
```suggestion
    potentials = [pot for (_, pot) in params_potential_pairs]
    params = [theta for (theta, _) in params_potential_pairs]
    return potentials, params
```